.github/workflows/lint.yml

```diff
@@ -0,0 +1,71 @@
+name: Lint
+
+on:
+  workflow_call:
+    inputs:
+      forceSkip:
+        default: "false"
+        type: string
+      directories:
+        required: true
+        description: "Directories to lint"
+        type: string
+      cmd:
+        required: true
+        description: "command to run"
+        type: string
+
+env:
+  DEBIAN_FRONTEND: "noninteractive"
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Task
+        uses: arduino/setup-task@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install prettier
+        run: task -p deps:install:prettier
+
+      - name: Lint
+        run: task -p lint:prettier
+
+  golangci-lint:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        directory: ${{ fromJson(inputs.directories) }}
+    steps:
+      - name: Install build dependencies
+        if: github.repository == 'werf/werf'
+        run: sudo apt-get install -y libbtrfs-dev
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: ${{ matrix.directory }}/go.mod
+
+      - name: Install Task
+        uses: arduino/setup-task@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install golangci-lint
+        run: task -p deps:install:golangci-lint
+
+      - name: Lint
+        working-directory: ${{ matrix.directory }}
+        run: ${{ inputs.cmd }}
```

Comment:
And if we have Taskfiles everywhere, then let's make it clear that "inputs.directory" is about where to find Taskfile.yaml, not just working dir. Maybe even better to just pass the Taskfile path via `-t` flag to the `task` executable instead.